### ISSUE TITLE
rename operator inappropriately removes the restriction

### ISSUE THREAD

**Issue by dimitri-yatsenko** [datajoint/datajoint-python#224]
```python
(rel & cond).proj('field')   # works correctly
(rel & cond).proj(field2='field')   # condition is no longer applied
```

**Comment by dimitri-yatsenko** [comment]
To reproduce in the unified pipeline:
```python:
from pipeline import common, experiment
(experiment.Session()*common.Animal() & 'session_date<"2016-03"').proj('session_date', setup='rig')
```

**Comment by dimitri-yatsenko** [comment]
Fixed in 85afac5
